### DISCUSSION TITLE
Welcome to RAGECOOP-V Discussions!

### DISCUSSION THREAD

**Discussion by EntenKoeniq** [RAGECOOP/RAGECOOP-V#43] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

**Comment by punkzappa007** [comment]
can you add autoconnect please.. thank you

**Comment by JABSOL** [comment]
How can you mod a rage coop server ?
